com.microsoft.mrtk.input/Simulation/Devices/ControllerSimulationSettings.cs

```diff
@@ -60,17 +61,57 @@ public InputActionReference ChangeNeutralPose
 
         [SerializeField]
         [Tooltip("The initial pose of the simulated hand")]
-        private SimulatedHandPose defaultPose = SimulatedHandPose.Neutral;
+        private GestureId defaultPose = GestureId.Open;
```

Comment:
It also looks like this now changes the default behavior from a `SimulatedHandPose` that was mapped to `GestureId.Flat` to using `GestureId.Open` as the default. Is that intentional?

Reply:
Yeah this is intentional to better match the MRTK2 behavior. It also much easier to poke with the Open Pose (finger pointering) compared to the Flat Pose (Hand spread out).

Reply:
Ooh! For some reason, I thought we had a work item a couple of months ago to change to `Flat` _for_ MRTK2 compatibility, but looking back at MRTK2 shows that this might have been backwards 😬

Reply:
Dave had intentionally changed this to the flat gesture @RogPodge . I agree that the poke-y gesture is nicer, but, Dave did make this change intentionally also for MRTK2 "parity". So....???

Reply:
@davidkline-ms any preference? I really really really prefer the poke gesture

Reply:
while i prefer the flat pose, since that is how physical hands will likely first be tracked, i am ok with either since there is the option to specify

Reply:
WIll be leaving in the open pose. Animation wise it'll also sneakily avoid the issues where the fingers bend backwards as it lerps into the pinch pose :)